src/app/message/list-entry/message-list-entry.component.styl

```diff
@@ -29,7 +29,7 @@ flag
   border: 3px solid transparent
 
 p[md-line]
-  white-space: normal
+  white-space: normal !important
```

Comment:
Just be more specific. Maybe `message-list-entry p[md-line]`. I don't know which is the wrapping element.